compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -168,6 +168,7 @@
     public final BXMLSubType xmlPIType = new BXMLSubType(TypeTags.XML_PI, Names.XML_PI);
     public final BXMLSubType xmlCommentType = new BXMLSubType(TypeTags.XML_COMMENT, Names.XML_COMMENT);
     public final BXMLSubType xmlTextType = new BXMLSubType(TypeTags.XML_TEXT, Names.XML_TEXT, Flags.READONLY);
+    public final BType xmlNeverType = new BXMLType(neverType,  null);
```

Comment:
Isn't this different from how we've modeled `xml<never>` at runtime? At runtime it is an xml:Text with never as the constraint, but at compile-time it is an xml with never as the constraint? Shall we stick to one approach?

Given that we don't have constraints for compile-time types, we'll probably have to change the runtime representation?